test/test_general_workflow.py

```diff
@@ -56,7 +56,6 @@ def test_lite_mode_block_hash():
     ],
     indirect=True,
 )
-@devnet_in_background(*PREDEPLOY_ACCOUNT_CLI_ARGS, "--lite-mode")
```

Comment:
Question. How come this is not needed anymore?

Reply:
It wasn't ever needed. So this is what we used to have:
```python
@pytest.mark.usefixtures("run_devnet_in_background")
@pytest.mark.parametrize(
    "run_devnet_in_background",
    [
        PREDEPLOY_ACCOUNT_CLI_ARGS,
        [*PREDEPLOY_ACCOUNT_CLI_ARGS, "--lite-mode"],
    ],
    indirect=True,
)
@devnet_in_background(*PREDEPLOY_ACCOUNT_CLI_ARGS, "--lite-mode")
```

If you look closely, you'll see it was spawning two devnets, and was even causing port collision, but the test passed regardless.